### PR TITLE
Update guest-access-checklist.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5319
Added important note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/guest-access-checklist.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ Watch this short video (5:31 minutes) to see how to turn on guest access through
 
 > [!VIDEO https://www.microsoft.com/videoplayer/embed/RE44NTr?autoplay=false]
 
+> [!IMPORTANT]
+> To get Teams to work with external/anonymous users, the **Anonymous users can join a meeting setting** must be turned on, which by default is off."
```
Comment (JohanFreelancer9) [inline Teams/guest-access-checklist.md:37]:
```suggestion
> To get Teams to work with external/anonymous users, the **Anonymous users can join a meeting setting**, which is off by default, must be turned on.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Hi @lindspea - Instead of the Important note, I added this as Step 7 and included steps for how to turn on the setting and a link to the Manage meeting settings in Teams topic.

**Comment by lindspea** [comment]
> Hi @lindspea - Instead of the Important note, I added this as Step 7 and included steps for how to turn on the setting and a link to the Manage meeting settings in Teams topic.

Thanks @LanaChin

**Pull request merged**